config/routes.rb

```diff
@@ -3,4 +3,9 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  # root "users#index"
+  get '/', to: 'users#index'
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:id/posts/:id', to: 'posts#show'
```

Comment:
**OPTIONAL** - You have done great so far, but I suggest you use a more recent method to configure your routes, this is cleaner and easier to understand. Please fix this 🙏 
```
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```